File: app/pojo/Item.java
package pojo;

public class Item {

    private String name;
    private int score;

    public Item(String name) {
        this.name = name;
        this.score = 0;
    }

    public String getName() {
        return name;
    }

    public int getScore() {
        return score;
    }

    public void increment() {
        this.score = this.score + 1;
    }

    public void decrement() {
        if (this.score > 0) {
            this.score = this.score - 1;
        }
    }
}

File: app/pojo/Item.scala
package pojo

class Item(var name: String, var score: Int) {

  def this(name: String ) {
    this(name, 0)
  }

  def increment(): Unit = this.score = this.score + 1

  def decrement(): Unit = if (this.score > 0) this.score = this.score - 1


}